dataland-frontend/tests/sharedUtils/components/UploadDocuments.ts

```diff
@@ -1,119 +1,58 @@
 import { TEST_PDF_FILE_BASEPATH } from "@sharedUtils/ConstantsForPdfs";
 
-export const uploadDocuments = {
```

Comment:
Or are those just moved to UploadReports?

Reply:
They got moved to UploadReports.ts because they are specific to the UploadReports component. And UploadDocuments here is supposed to only mirror the UploadDocumentsForm component.

Reply:
ok